fem/fe_coll.cpp

```diff
@@ -260,10 +260,30 @@ FiniteElementCollection *FiniteElementCollection::New(const char *name)
    }
    else if (!strncmp(name, "NURBS", 5))
    {
-      fec = new NURBSFECollection(atoi(name + 5));
+      int Order = atoi(name + 5);
+
+      if (Order < 10)
+      {
+         fec = new NURBSFECollection(Order);
+      }
+      else if (Order > 100)
```

Comment:
Document what this means

Reply:
First let me explain what I have done, than I will make a case how this could be simplified.

In the old NURBS implementation there was a single order, which was extracted from the FEC name. In the new NURBS implementation it was possible to have 3 distinct  orders, one for each direction. So in principle these are 3 numbers. However, I believe somewhere in the quadrature rules or shape routines, I saw a hardwired maximum of order 9. Either way, I encoded these 3 numbers in one number as px*100+py*10 +pz and used that in the name

This is statement is to make the distinction between single order NURBS (with one order) and multiple order NURBS (with 3 encoded orders).

The cleanest solution I believe, would be to not specify the order in the constructor.
 The specification of the NURBS order in the collection is redundant as the order gets set before the actual FE gets called. This happens int the LoadFE and LoadBE routines where patch and knot vector info are also set.

This will also solve some of the other issues mentioned.